config/schema.yml

```diff
@@ -44,6 +44,16 @@ propel:
     created_at: timestamp
     part_of: { type: integer, foreignTable: object, foreignReference: id, onDelete: setnull }
 
+  audit_log:
+    id: { type: integer, primaryKey: true, autoIncrement: true, inheritanceKey: false }
+    object_id: { type: integer, required: true, foreignTable: object, foreignReference: id, onDelete: cascade }
+    user_id: { type: integer, required: false, foreignTable: user, foreignReference: id, onDelete: setnull  }
+    user_name: varchar(255)
+    action_type_id: { type: integer, required: false, foreignTable: term, foreignReference: id, onDelete: setnull }
+    created_at: timestamp
+    _indexes:
+      1: [object_id]
```

Comment:
Are there searches on this table that may need an index other than by object_id?  Do we provide a search like "all changes by user"?  This table could get big.

Reply:
This index, and an index for user ID, actually get auto-generated so I've removed this from the schema and updated the migration.